src/v2/Components/Range.tsx

```diff
@@ -0,0 +1,132 @@
+import { themeGet } from "@styled-system/theme-get"
+import { Range as RCRange, RangeProps } from "rc-slider"
+import styled from "styled-components"
+
+const BORDER_RADIUS_BASE = 6
+export const RANGE_DOT_SIZE = 16
```

Comment:
Sorry for late feedback, but should `RANGE_DOT_SIZE` should be 32 to match Figma?

<img width="300" alt="Screen Shot 2022-03-13 at 12 10 50 PM" src="https://user-images.githubusercontent.com/140521/158068622-93a262d2-6462-4ee2-8204-a3910c0f7775.png">

Reply:
If we set the size to 32px, it will look too big. I discussed this point with Lois and showed this problem. She agreed that it was better to specify a smaller size

<img width="474" alt="image" src="https://user-images.githubusercontent.com/3513494/158163573-d788c311-cd77-48cd-bed5-d55ef05248d8.png">